src/skydb_v2_testplus.test.js

```diff
@@ -0,0 +1,241 @@
+const axios = require("axios");
```

Comment:
You can remove this file. We just removed these tests from `skynet-js` the other day.